source/hic_hal/maxim/max32625/gcc/startup_MAX32625.S

```diff
@@ -1,347 +0,0 @@
-/**
```

Comment:
Seems like I started  a "review" on github but I also should have submitted it at the end. Lesson learned... I should use the "comment" feature instead of the "start review"